datasetinsights/estimators/siam_mask.py

```diff
@@ -0,0 +1,2215 @@
+import glob
+import logging
+import math
+import random
+import time
+from os.path import join
+from pathlib import Path
+
+import cv2
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as FF
+from torch.autograd import Variable
+from torch.optim.lr_scheduler import _LRScheduler
+
+from datasetinsights.datasets import Dataset
+from datasetinsights.io.bbox import BBox2D
+from datasetinsights.io.loader import create_loader
+from datasetinsights.io.transforms import (
+    BlurImage,
+    CenterCrop,
+    GrayImage,
+    ScaleBBox,
+    ShiftBBox,
+)
+
+from .base import Estimator
+
+# from datasetinsights.evaluation_metrics.base import EvaluationMetric
+
+
+logger = logging.getLogger(__name__)
+
+
+def corner2center(corner):
```

Comment:
Should have docstring explanation of what is done.

Reply:
Also the formatting should match the canonical docstring format in other estimators. Roughly like:

"<description>

Args:
<indented list of args>

Returns:
<indented description of return value>